packages/core/lib/engine_http.js

```diff
@@ -350,8 +350,31 @@ HttpEngine.prototype.step = function step(requestSpec, ee, opts) {
 
         // Request.js -> Got.js translation
         if (params.qs) {
-          requestParams.searchParams = template(params.qs, context);
+          const searchParams = template(params.qs, context);
```

Comment:
Question: any reason to not just use `URLSearchParams()` here and avoid the rest of the new code, i.e.:

```js
const searchParams = new URLSearchParams(template(params.qs, context));
```